ADMIN.md

```diff
@@ -0,0 +1,35 @@
+# Guide for Lead Developer
+
+## The Visuals are served via this URL template
+
+URL Template: `https://chicagomaroon.github.io/data-visualizations/[PATH TO FILE]`
+
+URL Example: `https://chicagomaroon.github.io/data-visualizations/2023/thifting-growth/thifting-growth.min.html`
+
+## Commands
+
+-   `npm run lint`: Runs the formatter and linter in the repository and will let you know if any JavaScript faux pas were made in your code
```

Comment:
This will need to be updated when https://github.com/chicagomaroon/data-visualizations/pull/97 is merged.